### ISSUE TITLE
Embeded JSX in body expression gives indent lint error

### ISSUE THREAD

**Issue by gandm** [gandm/language-babel#133]
The code below gives alignment error for the traling })}
```jsx
        <div>
          <input type="text" 
            value={this.state.searchString} 
            onChange={this.handleChange} 
            placeholder="Type here" />
          <ul>
          { libraries.map(function(l){
            return <li>{l.name} <a href={l.url}>{l.url}></a></li>;
            }) }
          </ul>          
        </div>
```

**Comment by kevinsimper** [comment]
![language-babel](https://cloud.githubusercontent.com/assets/1126497/12535964/671f3b80-c296-11e5-8a77-66da413edc41.gif)

I have the same problem

**Comment by gandm** [comment]
@kevinsimper Until I can get around to fixing this you may have to disable auto-indentation either in the package settings ( globaly for all docs ) or toggle it on/off for the document using `language-babel:toggle-auto-indent-jsx`

**Comment by gandm** [comment]
@kevinsimper  This is now fixed in 2.15.0  If you want to try it enable auto Indenting  in the package settings. The first tag must be aligned manually, the remaining JSX should indent accordingly.

**Comment by kevinsimper** [comment]
:+1:
